app/src/main/java/org/simple/clinic/feature/Feature.kt

```diff
@@ -23,5 +23,6 @@ enum class Feature(
   ChangeLanguage(Build.VERSION.SDK_INT >= Build.VERSION_CODES.M),
   MLKitQrCodeScanner(true, "ml_kit_qr_code_scanner"),
   InstantSearch(true, "instant_search"),
-  OverdueCount(true, "overdue_count")
+  OverdueCount(true, "overdue_count"),
+  VillageTypeAhead(true, "village_type_ahead")
```

Comment:
Should the default value for this be set to `false` instead, given that we'll need to wait for the remote config sync to complete.

Reply:
Ah, good point. Will make the change.